wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ViewModel/ConversationListViewControllerViewModel.swift

```diff
@@ -115,6 +124,25 @@ extension ConversationListViewController {
 
         let getUserAccountImageSourceUseCase: any GetUserAccountImageSourceUseCaseProtocol
 
+        public var profileNotifications: ProfileNotifications? {
+            if userSession.selfUser.isTeamMember {
+                return nil
+            }
+            guard let apiVersion = BackendInfo.apiVersion,
+                  apiVersion > .v6 else {
+                return nil
+            }
+
+            let userDefaults = PrivateUserDefaults<UserDefaultsKey>(userID: userSession.selfUser.remoteIdentifier)
+            let value = userDefaults.object(forKey: .didViewSelfProfile)
```

Comment:
could be hidden under a convernience method:

var didViewSelfProfile: Bool {
get set...
}

Reply:
Moved into a separate object